OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -85,7 +85,7 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager)
 				}
 
 				var team = item.Get<LabelWidget>("TEAM");
-				var teamNumber = pp.PlayerReference.Playable ? ((client == null) ? 0 : client.Team) : pp.PlayerReference.Team;
+				var teamNumber = pp.PlayerReference.Playable ? client.Team : pp.PlayerReference.Team;
```

Comment:
`client` already gets de-referenced [earlier](https://github.com/OpenRA/OpenRA/pull/10905/files#diff-d371faa3132142c9bb8b07d783a5ea38R53), so if it was null, we'd never get here anyway.